visidata/features/describe.py

```diff
@@ -44,10 +45,6 @@ class DescribeSheet(ColumnsSheet):
             DescribeColumn('nulls',  type=vlen),
             DescribeColumn('distinct',type=vlen),
             DescribeColumn('mode',   type=str),
-            DescribeColumn('min',    type=str),
```

Comment:
Is there any functional reason to moving these into describe_aggrs?  I think this must be why min/max types were changed to `anytype` (and maybe they should not be `str` here in the original code), so maybe this is not necessary.

Reply:
I think the reason for doing this was  that`stdev` needed to be date-aware, so I couldn't use the standard `statistics.stdev` function. Instead I was using the `stdev` aggregator. While I was replacing standard library 
functions with aggregators, I moved `min`/`max`/etc too.